.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,67 +1,72 @@
-[Short description explaining the high-level reason for the pull request]
+_[Enter an explanation of what the pull request does and why.]_
 
-## Additions
 
--
-
-## Removals
-
--
+---
 
-## Changes
+_[Feel free to delete any checkboxes that are not applicable to this PR.]_
 
--
 
-## Testing
+## How to test this PR
 
 1.
 
+
 ## Screenshots
 
 
-## Notes
+## Notes and todos
 
 -
 
-## Todos
-
--
```

Comment:
In further looking for ways to condense the template, it seemed like these sections could be combined.

Reply:
Cool. I've often felt these overlap each other.